python/build-wheel-macos.sh

```diff
@@ -48,9 +48,11 @@ for ((i=0; i<${#PY_VERSIONS[@]}; ++i)); do
   PIP_CMD="$(dirname $PYTHON_EXE)/pip$PY_MM"
 
   pushd python
+    # Upgrade pip.
+    PATH=$MACPYTHON_PY_PREFIX/$PY_MM/bin:$PATH $PYTHON_EXE -m pip install --upgrade pip setuptools wheel
```

Comment:
Also, are you sure it's necessary to upgrade all of those?

Reply:
I'll try just pip